symfony/usr/local/share/symfony/symfony_functions.sh

```diff
@@ -58,12 +98,18 @@ do_database_build() {
 
 do_database_install() {
   do_database_schema_create
-  do_database_migrations_mark_done
+  if uses_symfony_doctrine_mode_migrations; then
+    do_database_migrations_mark_done
```

Comment:
ok, fair point. I did this as migrations could take much longer, but we could just rely on the s3 assets functionality to speed them up. Migrations may not even be ORM related, so running them in favour of the schema seems like the right idea when in migrations mode